test/coverage/datamodel/UMessageValidatorTest.cpp

```diff
@@ -353,22 +327,27 @@ TEST_F(TestUMessageValidator, ValidRpcResponse) {
 
 	{
 		// test common attributes for any message
-		auto attributes = fakeResponse(std::move(source_), std::move(sink_));
+		auto attributes = fakeResponse(source_, sink_);
 		testCommonAttributes(attributes);
 	}
 
 	{
 		// valid
-		auto attributes = fakeResponse(std::move(source_), std::move(sink_));
+		auto attributes = fakeResponse(source_, sink_);
 		auto umessage = build(attributes);
 		auto [valid, reason] = isValidRpcResponse(umessage);
 		EXPECT_TRUE(valid);
 		EXPECT_FALSE(reason.has_value());
+		// Only for debugging a test - should only happen if test is already
+		// failing and will always fail so that the reason message can be seen
+		if (reason) {
+			EXPECT_EQ("", message(*reason));
+		}
```

Comment:
KUDOS - nice "add" to the test - I'll keep this in mind for future tests, very useful especially when the expected pass doesn't pass.